ci-operator/step-registry/ipi/install/heterogeneous/ipi-install-heterogeneous-ref.yaml

```diff
@@ -1,6 +1,9 @@
 ref:
   as: ipi-install-heterogeneous
-  from: upi-installer
+  from_image:
```

Comment:
can't we use upi-installer (and import specific version one in the base_images) for cluster_claims?

Reply:
yes, it works, fixing